orchestration/dagster_orchestration/hca_orchestration/sensors.py

```diff
@@ -37,13 +39,20 @@ def generate_run_request(self, workflow: ExtendedArgoWorkflow) -> RunRequest:
                             "dataset_name": inflated_workflow.params_dict()['data-repo-name'].removeprefix("datarepo_"),
                         }
                     }
+                },
+                "resources": {
+                    "data_repo_client": {
+                        "config": {
+                            "api_url": "https://jade.datarepo-dev.broadinstitute.org"
```

Comment:
swap this to an env var fetch like the other config fields

Reply:
Is that a todo, and should it be a small-bite ticket?

Reply:
nope, already added to this pr